recipes/icon/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/env bash
+make Configure name=linux
+# point make to the real gcc
+export CC=x86_64-conda-linux-gnu-gcc
```

Comment:
Why?

Reply:
Here is the message if I don't specify the real gcc

```
make[1]: Entering directory '$SRC_DIR/src/common'
gcc -O -o infer infer.c
make[1]: gcc: No such file or directory
```

I will re-demonstrate this in CI with my next commit.

Reply:
here is the build that failed because `gcc` didn't point to anything:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=365071&view=logs&jobId=35904a6b-5404-55c5-26d9-f2fb250157bf&j=35904a6b-5404-55c5-26d9-f2fb250157bf&t=f869e631-70a1-522d-dae3-fa8b231d712d

Reply:
There were lots of changes after that. Can you please remove this line?

Reply:
Still [fails as expected](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=370926&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=637) in the Azure pipeline.  I will revert but make it conditional upon the absence of `gcc`